README.md

```diff
@@ -34,6 +34,7 @@ Run(Args, targets);
 | List targets                  | `{runner} .\build.csx -T`                                                           |
 | List targets and dependencies | `{runner} .\build.csx -D`                                                           |
 | Dry run                       | Append `-n`, e.g. `{runner} .\build.csx -n` or `{runner} .\build.csx targetname -n` |
+| Skip running depenedencies    | Append `-s`, e.g. `{runner} .\build.csx -s` or `{runner} .\build.csx targetname -s` |

```

Comment:
The typo is still here 😉

Reply:
HA! I was looking at the one on the right side of the table!